health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/AppConfigurationParentLayer.js

```
@@ -53,78 +53,63 @@ const dispatcher = (state, action) => {
+  // Try to find a matching field type from master data
+  const matched = fieldTypeMasterData.find((item) => {
+    // Match both type and format from metadata
+    return item?.metadata?.type === field.type && item?.metadata?.format === field.format;
+  });
+
+  if (!matched) {
+    return { type: "textInput" };
   }
+
+  // Start with the fieldType as type
+  let result = { type: matched.fieldType, appType: matched.type };
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'result' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)